int32 id
geometry_msgs/Pose pose
float64 meanError
bool isValid
int16 params
